README.ru.md

```diff
@@ -52,11 +52,11 @@
 - [Содержание](#содержание)
 - [Вступление](#вступление)
 - [Особенности](#особенности)
-    - [Roadmap](#roadmap)
+  - [Roadmap](#roadmap)
 - [Быстрый старт](#быстрый-старт)
 - [Спонсоры](#спонсоры)
 - [FAQ](#faq)
-- [График звёздочек](#график-звёздочек-репозитория-относительно-времени)
```

Comment:
Why this change?

Reply:
it was a correction from my editor!! I just fixed it.